quiz.html

```diff
@@ -11,33 +11,38 @@
 		</header>
 		
 		<div class="wrapper">
-
-			<!-- This div is for the skin type quiz and is different to the other pages -->
-			<div class="quizbox">
-				<h1>Question 1 heading</h1>
-				<h2>1 / 5</h2>
-				<p>Question related images and text will go here</p>
-				<hr color="white" width="80%" size="3">
-
-				<h1>Question 2 heading</h1>
-				<h2>2 / 5</h2>
-				<p>Question related images and text will go here</p>
-				<hr color="white" width="80%" size="3">
-
-				<h1>Question 3 heading</h1>
-				<h2>3 / 5</h2>
-				<p>Question related images and text will go here</p>
-				<hr color="white" width="80%" size="3">
-
-				<h1>Question 4 heading</h1>
-				<h2>4 / 5</h2>
-				<p>Question related images and text will go here</p>
-				<hr color="white" width="80%" size="3">
-
-				<h1>Question 5 heading</h1>
-				<h2>5 / 5</h2>
-				<p>Question related images and text will go here</p>
-
+			<div class="quiz-page">
+				<h1>Skin Type Quiz</h1>

```

Comment:
In this layout, I included the heading as seen in layout 2's design but kept the overall quiz layout the same as layout 1.